x-pack/plugin/transform/src/main/java/org/elasticsearch/xpack/transform/persistence/TransformInternalIndex.java

```diff
@@ -413,14 +441,21 @@ protected static void createLatestVersionedIndexIfRequired(
             CreateIndexRequest request = new CreateIndexRequest(TransformInternalIndexConstants.LATEST_INDEX_VERSIONED_NAME)
                 .settings(settings())
                 .mapping(mappings())
-                .origin(TRANSFORM_ORIGIN);
+                .origin(TRANSFORM_ORIGIN)
+                .waitForActiveShards(ActiveShardCount.ALL);
```

Comment:
I think having `ALL` here is fine as the index has our original settings, which doesn't include replicas (I think).